src/ORM/Query.php

```diff
@@ -33,6 +33,7 @@
  *
  * @see \Cake\Collection\CollectionInterface For a full description of the collection methods supported by this class
  * @method \Cake\Collection\CollectionInterface each(callable $c) Passes each of the query results to the callable
+ * @method \Cake\Collection\CollectionInterface sortBy($callback, $dir = SORT_DESC, $type = \SORT_NUMERIC) Sorts the query with the callback
```

Comment:
Why the leading blacklash at SORT_NUMERIC?

Reply:
Good question. I did not drill that point but it's exactly the signature of this method in CollectionInterface.
I see in CollectionTrait/sortBy(), there is also a backslash on the default value for the second argument.
So, it could be better even to put both, or to suppress both.
- To put : I think that in a pure programming rules, it could be better to put the slashes to be sure to get the php global namespace. Thus, if someone redefine this constant in the local namespace, the results would be uncertain. 
- To suppres : I think it's not necessary to use backslash in this context because it's almost impossible that someone would redefine that kind of constant, and it results with less readable code, and all along the cakephp program the use of backslash for SORT_ASC (for example) is very different sometimes with backslash and not, but more often without.
I you and others agree to suppress, I can fix that for sortBy method in CollectionInterface and in CollectionTrait.

Reply:
Sounds like a snowflake accident, as we dont do that anywhere in code - and probably shouldnt as it would limit the desired possibility of overwriting in certain cases.
So best to remove this and keep the code consistent here.

Reply:
`"PS: why so many comments? Are you able to squash them into a single one afterwards?"`

I guess it's commits (not comments). If yes, you are right I will try to do it (try : because I'm a beginner on Git and Github, and it's why there's so many commits !).

`"...as we dont do that anywhere in code"`
If you look for SORT_ASC or SORT_NUMERIC you will find few backslahes...

Ok, I will purpose the fix...

Reply:
The leading slashes were added to the actual code because we ran into build issues in travis and adding a slash made the problems go away.

Reply:
👍